extlinks/links/migrations/0015_remove_linkevent_url.py

```diff
@@ -0,0 +1,17 @@
+# Generated by Django 4.2.14 on 2024-09-27 18:12
```

Comment:
We actually need to keep this field in the model for it to be available for data migration. We'll need to remove it in a followup patch.